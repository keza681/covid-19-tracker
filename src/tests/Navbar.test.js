import React from 'react';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import configureStore from '../redux/store';
import Nav from '../components/Navbar';

test('router test', () => {
  const tree = renderer
    .create(
      <Provider store={configureStore}>
        <BrowserRouter>
          <Nav />
        </BrowserRouter>
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
