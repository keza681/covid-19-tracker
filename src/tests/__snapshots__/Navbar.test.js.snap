// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`router test 1`] = `
<nav>
  <section
    className="navLinks"
  >
    <div>
      <a
        aria-current="page"
        className="active"
        href="/"
        onClick={[Function]}
      >
        <img
          alt="back"
          src="https://img.icons8.com/ios-filled/24/ffffff/back.png"
        />
      </a>
    </div>
    <div
      className="navIcons"
    >
      <img
        alt="mic"
        className="navIcon"
        src="https://img.icons8.com/fluency-systems-filled/24/ffffff/microphone.png"
      />
      <img
        alt="settings"
        className="navIcon"
        src="https://img.icons8.com/external-tanah-basah-glyph-tanah-basah/24/ffffff/external-setting-essentials-pack-tanah-basah-glyph-tanah-basah.png"
      />
    </div>
  </section>
</nav>
`;
